docs/source/user/threading-layer.rst

```diff
@@ -59,6 +59,21 @@ choosing a threading layer, the first is by selecting a threading layer that is
 safe under various forms of parallel execution, the second is through explicit
 selection via the threading layer name (e.g. ``tbb``).
 
+Setting the threading layer priority
+------------------------------------
+
+By default, The threading layers are searched in the order of
+'tbb', 'omp', and 'workqueue'.
+If you want to let Numba choice the threading layer
```

Comment:
```suggestion
If you want to let Numba choose the threading layer
```

Reply:
GitHub marked this as outdated. You already made a suggestion in b11cf88.